sematic/db/queries.py

```diff
@@ -159,20 +158,23 @@ def get_run_status_details(
     """
     with db().get_session() as session:
         query_results = (
```

Comment:
The other results variable is called `jobs`, so please rename this one to differentiate them.
```suggestion
        runs = (
```